broker/src/main/java/org/apache/rocketmq/broker/processor/AdminBrokerProcessor.java

```diff
@@ -267,9 +268,16 @@ private synchronized RemotingCommand updateAndCreateTopic(ChannelHandlerContext
         topicConfig.setTopicFilterType(requestHeader.getTopicFilterTypeEnum());
         topicConfig.setPerm(requestHeader.getPerm());
         topicConfig.setTopicSysFlag(requestHeader.getTopicSysFlag() == null ? 0 : requestHeader.getTopicSysFlag());
+        topicConfig.setExpirationTime(requestHeader.getExpireTime());
```

Comment:
Should topic expireTime limit the maximum time? If expireTime is greater than the maximum time for commitlog storage, expireTime is meaningless?